napari/viewer.py

```diff
@@ -45,3 +45,9 @@ def __init__(self, title='napari', ndisplay=2, order=None):
         self.window = Window(qt_viewer)
         self.screenshot = self.window.qt_viewer.screenshot
         self.update_console = self.window.qt_viewer.console.push
+
+    @staticmethod
+    def update(func, *args, **kwargs):
+        t = QtUpdateUI(func, args, kwargs)
```

Comment:
```suggestion
        t = QtUpdateUI(func, *args, **kwargs)
```

Reply:
I am willing to accept this suggestion but what is the rational behind? I want to learn. As far as i observed it is not needed indeed?

Reply:
I think it's a coincidence that it worked because your current code didn't have keyword-only arguments, and had exactly two arguments. `QtUpdateUI(func, args, kwargs)` was passing two positional arguments to `__init__`, both of which would have ended up in the `.args` attribute. Since this particular function could take two positional arguments, it was working fine, but it wasn't doing what you thought it was doing.